### DISCUSSION TITLE
When I buy the Pro version, will it be compatible with Janet?

### DISCUSSION THREAD

**Discussion by fuxoft** [nesbox/TIC-80#2273] [Q&A] [answered]
I understand that TIC 80 now supports Janet and I'd like to buy the Pro version and use it to teach Janet to two people who want to learn programming.

But it seems that Janet support was only added in v 1.1 and the "Pro" version is v 1.0 (as seen at [https://nesbox.itch.io/tic80](https://nesbox.itch.io/tic80)) Is this correct?

**Comment by nesbox** [comment, marked as answer]
Yes, PRO hasn't released yet, I think it will be released on the next week.

**Reply by fuxoft** [reply]
OK, I've bought it and I'll be waiting for the new Pro version...

**Reply by sogaiu** [reply]
FWIW, the [latest release](https://github.com/nesbox/TIC-80/releases/tag/v1.1.2729) has worked with Janet in my limited testing.

**Reply by fuxoft** [reply]
But that latest release is not available in the Pro version yet so I can't use it with external code editor. That was the point of my original question - the Pro version is currently only available as v 1.0 and is not compatible with Janet, which seems to be confirmed by the author above.
